### DISCUSSION TITLE
Utm se not being able to open ubuntu

### DISCUSSION THREAD

**Discussion by GGGoradia** [utmapp/UTM#6610] [Linux]
I downloaded the iso file the options till try or install ubuntu is coming but after that only black screen is being visible 

**Configuration (required)**
* UTM Version: utm se
* OS Version: ubuntu 22.04
* Device Model: ipad 10th gen
* Is it jailbroken (name jailbreak used)?  No
* How did you install UTM? Appstore

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open the Settings app and browse to `Privacy -> Analytics & Improvements -> Analytics Data` and find the latest entry for UTM. You should export the text and attach it here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the bottom of the configuration options (or at the top of the `QEMU` page) is `Debug Log`. Turn it on and save the VM. After you experience the issue, quit the VM and re-launch UTM. Open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by LostInCompilation** [comment]
The booting probably takes an hour because of emulation. Also iPad 10 isn't the fastest, Ubuntu is huge. Are you using x86 or ARM ubuntu?

**Comment by GGGoradia** [comment]
So which os should i run on my ipad ?? Is there any alternative? Also im using arm64 version

**Comment by LostInCompilation** [comment]
You can check out some [light-weight Linux distros](https://en.wikipedia.org/wiki/Light-weight_Linux_distribution). Depends on what you want to do with the OS, if you need a GUI and so on...

**Comment by GGGoradia** [comment]
Oh okay yes i need the gui will distors be fine then for my ipad?
